routes.js

```diff
@@ -47,5 +48,9 @@ router.patch('/event-question', auth, (req, res) => eventQuestionController.upda
 router.post('/event-question/response', auth, (req, res) => eventQuestionController.addResponseToEvent(req, res));
 router.delete('/event-question/response/:eventQuestionId/:responseId', auth, (req, res) => eventQuestionController.deleteResponse(req, res));
 
+// ResponseType routes
+router.get('/response-type', (req, resp) => responseTypeController.getAll(req, resp));
```

Comment:
Add authentincation for all routes of the api, not a single part of the api should be exposed to the public, even if its is just a response type object with no big value.